drivers/wifi/nrf700x/osal/os_if/src/osal.c

```diff
@@ -200,7 +200,7 @@ void nrf_wifi_osal_spinlock_irq_rel(struct nrf_wifi_osal_priv *opriv,
 }
 
 
-#ifdef CONFIG_WIFI_NRF700X_LOG_LEVEL_DBG
+#if CONFIG_WIFI_NRF700X_LOG_LEVEL >= CONFIG_WIFI_NRF700X_LOG_LEVEL_DBG
```

Comment:
Is it correct? Isn't CONFIG_WIFI_NRF700X_LOG_LEVEL_DBG a boolean like in other modules? I think you should rather compare the log level to `LOG_LEVEL_XXX` constants from `log.h`.

Reply:
Oops, you are right its a boolean choice, but I don't think we have a per-module logging levels?

Reply:
Except for using the number directly (`4` in this case), I don't see a way here, WDYT?

Reply:
ok, thanks, it should.